e2e_test.ts

```diff
@@ -34,55 +34,119 @@ import options from "./fresh.config.ts";
  */
 const handler = await createHandler(manifest, options);
 
-function assertResponseNotFound(resp: Response) {
+function assertResponseAccepted(resp: Response) {
+  assert(resp.ok);
+  assertEquals(resp.body, null);
+  assertEquals(resp.status, Status.Accepted);
+}
+
+function assertResponseBadRequest(resp: Response) {
   assertFalse(resp.ok);
-  assertEquals(resp.status, Status.NotFound);
+  assertEquals(
+    resp.headers.get("content-type"),
+    "text/plain;charset=UTF-8",
+  );
+  assertEquals(resp.status, Status.BadRequest);
+}
+
+function assertResponseCreated(resp: Response) {
+  assert(resp.ok);
+  assertEquals(resp.body, null);
+  assertEquals(resp.status, Status.Created);
+}
+
+function assertResponseHtml(resp: Response) {
+  assert(resp.ok);
+  assertInstanceOf(resp.body, ReadableStream);
+  assertEquals(
+    resp.headers.get("content-type"),
+    "text/html; charset=utf-8",
+  );
+  assertEquals(resp.status, Status.OK);
```

Comment:
```suggestion
function assertHtml(resp: Response) {
  assertInstanceOf(resp.body, ReadableStream);
  assertEquals(
    resp.headers.get("content-type"),
    "text/html; charset=utf-8",
  );
```
This does a little more than it needs to.

Reply:
applied as suggested!